capture-traffic/capture-network-traffic.md

```diff
@@ -0,0 +1,181 @@
+
+In nature, the network capturing mode is a more powerful way to capture network traffic on a lower level than an HTTPS(S) proxy works (which is how Fiddler works in the **System Capturing** mode or all other available capturing modes). 
+
+The benefits of the network capturing mode are as follows:
+
+- Works on a lower level compared to HTTP(S) proxies.
+
+- Captures all HTTP(S) traffic from the active network adapter. 
+
+- No limitations related to frameworks, applications, operating systems, and other configuration specifics that you usually must handle when using an HTTP(S) proxy.
+
+- Allows you to control the IP addresses and processes to monitor seamlessly.
+
+## Prerequisites & Limitations
+
+### Prerequisites
+
+You must meet the following prerequisites to use the network capturing mode.
+
+- Installed the latest version of Fiddler Everywhere.
+
+- Administrative privileges to install/uninstall the network extension on macOS.
+
+
+### Limitations
+
+The **Network Capture** mode is in Beta state and, currently, has some known limitations as follows:
+
+- The network extension that enables the network capturing mode is available only on the macOS version of Fiddler Everywhere. The development for the counterpart Windows & Linux features is ongoing.
+
+- The network capturing mode captures IPv4 traffic, while IPv6 is not yet supported.
+
+- The  network capturing mode is incompatible with most VPN tools.
+
+    >tip Some VPN tools will automatically close the VPN connection is a third-party network extension is enabled. You can try to workaround that by starting Fiddler Everywhere and enabling the network capturing mode before making a connection to the VPN.
+
+- The **HTTPS > Connections > Bypass Fiddler for URLs that starts with:** option is incompatible with the **Network Capture** mode. Alternatively, you can create a Fiddler rule that executes the **Do Not Decrypt** action on the desired domains.
+
+- The network capturing mode can capture only outbound traffic. Currently, inbound traffic from remote device is not captured. For such cases, use our alternative capturing modes for [Android]({%slug capture-mobile-android-traffic%}), [iOS]({%slug capture-mobile-ios-traffic%}), or [remote PCs]({%slug fiddler-capture-other-computers%}).
+
+
+## Using the Network Capturing Mode
+
+The network capturing mode is accessible in the Fiddler's **Live Traffic** menu options through the **Network Capture** option.
+
+![the "Network Capture" feature](../images/livetraffic/websessions/websessions-toolbar-network-capture-beta.png)
+
+Once the **Network Capture** is enabled, you can capture all HTTP(S) traffic through the active network adapter. As noted in the benefits section, the main perk of using the network capturing mode is that you don't need to change the operating system proxy or set the client's proxy. That lets you quickly capture traffic from processes that otherwise have trouble using HTTP(S) proxies.
```

Comment:
```suggestion
Once the **Network Capture** is enabled, you can capture all TCP traffic. As noted in the benefits section, the main perk of using the network capturing mode is that you don't need to change the operating system proxy or set the client's proxy. That lets you quickly capture traffic from processes that otherwise have trouble using HTTP(S) proxies.
```